GiTinder/Services/UserServices.cs

```diff
@@ -12,6 +27,46 @@ public UserServices(GiTinderContext context)
         {
             _context = context;
         }
+<<<<<<< HEAD
+        //async method is used everytime something should take long time like for requesting for data await allows certain line to run independently.
+        //if user doesn't exist in github then it shouldnt respond
```

Comment:
@Riceqrisp do we really need the comments?

Reply:
Comments erased